components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -666,6 +666,8 @@ func NewProvisioningProcessingQueue(ctx context.Context, provisionManager *proce
 		{
 			stage: createRuntimeStageName,
 			step:  provisioning.NewOverridesFromSecretsAndConfigStep(db.Operations(), runtimeOverrides, runtimeVerConfigurator),
+			// Preview plan does not calls Reconciler so it does not need overrides
```

Comment:
nit: in that case, I think we can skip BTP Operator related steps as well here and in the `update` queue.

Reply:
BTP operator steps must be run - we will provide thhe secret directly to SKR

Reply:
and just to make sure, is it ok to run these reconciler interactions in the update queue? or do we want to also skip them?
https://github.com/kyma-project/control-plane/blob/e3b5f1eab946516db36765a7bf8a56c81430f0e5/components/kyma-environment-broker/cmd/broker/main.go#L767-L774

Reply:
for the record, @piotrmiskiewicz just explained to me it's actually skipped for preview plan, just [here](https://github.com/kyma-project/control-plane/pull/2179/files#diff-84e967d32a4a3ed65022482cc8724cfd81b0b8e902c2a835775a40e009e04b49R10), I completely missed that